projects/storefrontlib/src/layout/direction/config/direction.config.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { Config } from '@spartacus/core';
+import { Direction } from './direction.model';
+
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class DirectionConfig {
+  /**
+   * The layout direction can be used to setup the html with an instruction for text and layout direction.
+   * Besides the default html direction modes (auto, ltr and rtl), we also support an auto detect mode, which
```

Comment:
If I understood well, we won't support `auto`
```suggestion
   * Besides the default html direction modes (ltr and rtl), we also support an auto detect mode, which
```

Reply:
correct, I'll reword